images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -1,5 +1,5 @@
 function Get-AnsibleVersion {
-    $ansibleVersion = ansible --version | Select-Object -First 1 | Take-OutputPart -Part 1
+    $ansibleVersion = (ansible --version)[0] -replace "[^\d.]"
     return "Ansible $ansibleVersion"
```

Comment:
What do you think does it make sense to rename it to Ansible-core for visibility?

Reply:
I prefer just Ansible without core. Maybe it makes sense to add src, like that Ansible x.x.x (pipx: ansible-core). Let's vote: @maxim-lobanov , @AlenaSviridenko, @MaksimZhukov

Reply:
I prefer Ansible without core too.

Reply:
But it is not ansible itself, isn't it? it is the ansible-core version